test/common.js

```diff
@@ -250,6 +250,28 @@ exports.childShouldThrowAndAbort = function() {
   });
 };
 
+
+// This test makes sure that when using --abort-on-uncaught-exception and
+// when throwing an error from within a domain that has an error handler
+// setup, the process _does not_ abort.
+
+exports.childShouldNotThrowAndAbort = () => {
+  let testCmd = '';
+  if (!exports.isWindows) {
+    testCmd += 'ulimit -c 0 && ';
+  }
+
+  testCmd += `${process.argv[0]} --abort-on-uncaught-exception`;
+  testCmd += ` ${process.argv[1]} child`;
+
+  const child = child_process.exec(testCmd);
+  child.on('exit', (code, signal) => {
+    const errorMsg = `Test should have exited with exit code 0 
+    but insteand exited with code ${code} and signal ${signal}`;
```

Comment:
The multiline string literal is likely going to be problematic because the additional whitespace is preserved. Can you make this the following instead:

```
const errorMessage = 'Test should have exited with exit code 0 but instead ' +
                                    `exited with ${code} and signal ${signal}`;
```

Generally speaking we do not use multiline string literals in core code